### ISSUE TITLE
customActionHandler not triggered on 2.0.0 and current master branch

### ISSUE THREAD

**Issue by galexrt** [Simplix-Softworks/Cirrus#14]
When using current `master` branch / snapshot, a `customActionHandler()` is not triggered for empty slots due to the item being `null`:

* Click event is returned if the slot/item is empty before it is sent:
https://github.com/Simplix-Softworks/Cirrus/blob/CirrusV2/cirrus-bungeecord/src/main/java/dev/simplix/cirrus/bungeecord/protocolize/ProtocolizeMenuBuilder.java#L116-L118
* `new Click()` does not allow the item to be Null (has `lombok.NotNull` annotation)

My current "fix" for being able to utilize a `customActionHandler()` for my menus is the following "patch":

```diff
diff --git a/cirrus-bungeecord/src/main/java/dev/simplix/cirrus/bungeecord/protocolize/ProtocolizeMenuBuilder.java b/cirrus-bungeecord/src/main/java/dev/simplix/cirrus/bungeecord/protocolize/ProtocolizeMenuBuilder.java
index fbc5d2a..7fd2426 100644
--- a/cirrus-bungeecord/src/main/java/dev/simplix/cirrus/bungeecord/protocolize/ProtocolizeMenuBuilder.java
+++ b/cirrus-bungeecord/src/main/java/dev/simplix/cirrus/bungeecord/protocolize/ProtocolizeMenuBuilder.java
@@ -142,9 +142,11 @@ public class ProtocolizeMenuBuilder implements MenuBuilder {
             if (inventoryClick.player() == null) {
                 return;
             }
+            /*
             if (inventoryClick.clickedItem() == null) {
                 return;
             }
+            */
             Inventory i = inventoryClick.inventory();
             if (i == null) {
                 return;
diff --git a/cirrus-common/src/main/java/dev/simplix/cirrus/common/model/Click.java b/cirrus-common/src/main/java/dev/simplix/cirrus/common/model/Click.java
index 9ea166f..a85fb46 100644
--- a/cirrus-common/src/main/java/dev/simplix/cirrus/common/model/Click.java
+++ b/cirrus-common/src/main/java/dev/simplix/cirrus/common/model/Click.java
@@ -6,6 +6,9 @@ import dev.simplix.cirrus.common.business.PlayerWrapper;
 import dev.simplix.cirrus.common.menu.Menu;
 import dev.simplix.protocolize.api.ClickType;
 import java.util.List;
+
+import org.jetbrains.annotations.Nullable;
+
 import lombok.NonNull;

 /**
@@ -21,7 +24,7 @@ public class Click {
     public Click(
             @NonNull ClickType clickType,
             @NonNull Menu clickedMenu,
-            @NonNull InventoryMenuItemWrapper clickedItem,
+            @Nullable InventoryMenuItemWrapper clickedItem,
             int slot) {
         this.clickType = clickType;
         this.clickedMenu = clickedMenu;
```

**Comment by KotlinFactory** [comment]
Hey,
this was indeed unintended, if you have got a fix for this feel free to PR it.

Regards,
Leonhard

**Comment by KotlinFactory** [comment]
Closed by #16
